cli/azd/pkg/tools/azcli/deployments.go

```diff
@@ -75,3 +78,127 @@ func (cli *azCli) createDeploymentsClient(
 
 	return client, nil
 }
+
+func (cli *azCli) DeployToSubscription(
+	ctx context.Context, subscriptionId, deploymentName, compiledBicep, parametersFile, location string) (
```

Comment:
Could we rename `compliedBicep` to `deploymentTemplate` or something (and if we create an `ArmTemplate` type, use that here)?  I think that better captures what is going with this change.

I'd love to not use "bicep" here if possible because when we go to support ARM deployments, we'd still want to use this.

Reply:
I saw `compiled` and `bicepTemplate` around there.  That's how I ended up with `compiledBicep` :)

![image](https://user-images.githubusercontent.com/24213737/196604716-d5cffa09-0676-4e4e-b760-d2d5ef5034c7.png)

We have a `BicepTemplate` struct, which is weird because it looks like an Arm template but withou the `resources` field